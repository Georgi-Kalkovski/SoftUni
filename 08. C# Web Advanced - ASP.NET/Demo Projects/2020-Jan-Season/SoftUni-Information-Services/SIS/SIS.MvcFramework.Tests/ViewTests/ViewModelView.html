<h1>Hello, @Model.Name</h1>
<p>It is @Model.Year year!</p>
<ul>
    @foreach(var num in Model.Numbers)
    {
        <li>@Math.Log10(num)</li>
    }
</ul>
<p>@User</p>