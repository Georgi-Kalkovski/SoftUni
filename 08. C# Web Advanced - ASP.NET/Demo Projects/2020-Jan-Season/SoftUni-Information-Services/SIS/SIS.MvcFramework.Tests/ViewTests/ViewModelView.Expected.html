<h1>Hello, Niki</h1>
<p>It is 2020 year!</p>
<ul>
        <li>0</li>
        <li>1</li>
        <li>2</li>
        <li>3</li>
        <li>4</li>
</ul>
<p>123</p>
